src/style.css

```diff
@@ -0,0 +1,71 @@
+.container {
+  height: 160px;
+  width: 140px;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-left: 22px;
+  margin-right: 22px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 > :nth-child(n) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--1 > :first-child {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
group

Reply:
nice ... thank you for this advice. 
applied !!!